src/main/java/com/server/domain/user/service/UserService.java

```diff
@@ -57,13 +64,38 @@ public GetUserOutDto getUserWithoutPersonalInfo(String username) {
         return userMapper.toGetUserOutDto(user);
     }
 
-    public User updateEmail(User user, String email) {
+    public User updateEmail(HttpServletRequest request, String email) {
+        String username = jwtService.extractUsernameFromToken(request)
+                .orElseThrow(() -> new AuthException(AuthErrorCode.INVALID_ACCESS_TOKEN));
+
+        // username으로 찾은 user 반환
+        User user = getUserWithPersonalInfo(username);
+
         user.updateEmail(email);
         return userRepository.save(user);
 
     }
 
-    public void deleteUser(User user) {
+    public String  deleteUser(HttpServletRequest request) {
```

Comment:
```suggestion
    public String deleteUser(HttpServletRequest request) {
```

공백이 2개 존재 하여 수정부탁드립니다.